corehq/toggles.py

```diff
@@ -636,17 +636,16 @@ def _ensure_valid_randomness(randomness):
 
 EXTENSION_CASES_SYNC_ENABLED = StaticToggle(
     'extension_sync',
-    'Enikshay/L10K: Enable extension syncing',
+    'L10K: Enable extension syncing',
```

Comment:
Can you check with Andrea(?) that this is accurate? I thought extensions cases were a solutions thing now in certain circumstances. 

At the least ICDS should be added here because its also used

Reply:
What is the reason behind keeping the name of the project in the name?
looking at the domains its enabled for should be enough to know what project is using it. Putting a project name in the name would always need us to come back and do such PRs.

Reply:
Checked with Andrea and switched to solutions limited use.

@mkangia I agree, having project name in flag names isn't especially useful, plus it gets misleading. Removed L10K from this name. But I'm not going through and removing ICDS from all the flag names that use it.